lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -142,15 +142,14 @@ def instructor_dashboard_2(request, course_id):
     if course_mode_has_price and (access['finance_admin'] or access['sales_admin']):
         sections.append(_section_e_commerce(course, access, paid_modes[0], is_white_label, is_white_label))
 
-    # Gate access to Proctoring tab
-    # only global staff (user.is_staff) is allowed to see this tab
-    can_see_proctoring = (
-        settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False) and
-        course.enable_proctored_exams and
-        request.user.is_staff
+    # Gate access to Special Exam tab depending if either timed exams or proctored exams
+    # are enabled in the course
+    can_see_special_exams = (
+        ((course.enable_proctored_exams and request.user.is_staff) or course.enable_timed_exams) and
+        settings.FEATURES['ENABLE_SPECIAL_EXAMS']
```

Comment:
Other parts of this code are using settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False) to provide a default value.